openbas-front/src/utils/Localization.js

```diff
@@ -1165,46 +1180,47 @@ const i18n = {
       x86_64: 'x86_64',
       arm64: 'arm64',
-      'Invalid IP addresses': 'Invalid IP addresses',
-      'Invalid MAC addresses': 'Invalid MAC addresses',
-      'Create a new security platform': 'Create a new security platform',
+      'Invalid IP addresses': 'Adresses IP invalides',
+      'Invalid MAC addresses': 'Adresses MAC invalides',
+      'Create a new security platform': 'Créer une nouvelle plateforme de sécurité',
       EDR: 'EDR',
       XDR: 'XDR',
       SIEM: 'SIEM',
       SOAR: 'SOAR',
       NDR: 'NDR',
       ISPM: 'ISPM',
-      'Logo light': 'Logo light',
-      'Logo dark': 'Logo dark',
-      'Update the security platform': 'Update the security platform',
-      'Security Platforms': 'Security Platforms',
-      'Content should not be empty': 'Content should not be empty',
-      Timezone: 'Timezone',
-      Log: 'Log',
-      'Your file should be a XLS': 'Your file should be a XLS',
-      'Launch import': 'Launch import',
-      'Show timeline': 'Show timeline',
-      'Selected payload': 'Selected payload',
-      Prevention: 'Prevention',
-      'Start date should be at least today': 'Start date should be at least today',
-      'Update the xls mapper': 'Update the xls mapper',
-      'Data ingestion': 'Data ingestion',
-      'XLS mappers': 'XLS mappers',
-      'Rule attributes columns': 'Rule attributes columns',
-      'Testing XLS mapper': 'Testing XLS mapper',
-      Policies: 'Policies',
-      'Login messages': 'Login messages',
-      'Export this list': 'Export this list',
-      'Should be a valid XLS file': 'Should be a valid XLS file',
-      'This file is not in the specified format': 'This file is not in the specified format',
-      'This file is too large': 'This file is too large',
-      'Associated file': 'Associated file',
-      'Select your file': 'Select your file',
-      'Add document': 'Add document',
-      'I have read and comply with the above statement': 'I have read and comply with the above statement',
-      'Date should be at least today': 'Date should be at least today',
-      'Security platform': 'Security platform',
-      'Security platforms': 'Security platforms',
+      'Logo light': 'Logo clair',
+      'Logo dark': 'Logo sombre',
+      'Update the security platform': 'Mettre à jour la plateforme de sécurité',
+      'Security Platforms': 'Plateformes de sécurité',
+      'Content should not be empty': 'Le contenu ne doit pas être vide',
+      Timezone: 'Fuseau horaire',
+      Log: 'Journal',
+      'Your file should be a XLS': 'Votre fichier doit être un XLS',
+      'Launch import': "Lancer l'importation",
+      'Show timeline': 'Afficher la chronologie',
+      'Selected payload': 'Charge utile sélectionnée',
+      Prevention: 'Prévention',
+      'Start date should be at least today': "La date de début doit être au moins aujourd'hui",
+      'Update the xls mapper': 'Mettre à jour le mapper xls',
```

Comment:
I know it's not linked to your PR but :
```suggestion
      'Update the xls mapper': 'Mettre à jour le mappeur xls',
```